trino/trino.go

```diff
@@ -1139,7 +1139,14 @@ func (qr *driverRows) fetch() error {
 				return nil
 			}
 		case err = <-qr.stmt.errors:
-			if err == context.Canceled {
+			if err == nil {
+				// Channel was closed, which means the statement
+				// or rows were closed.
+				// Previously this would panic with an index out
+				// of bounds, now we return io.EOF, see
+				// https://github.com/trinodb/trino-go-client/issues/65.
```

Comment:
Don't refer to the past, this is what git commit history is for.

Reply:
You still have a comment here that refers to past behavior, it is not needed.

Reply:
I though you were only referring to the linking of the issue, removed the second part of the comment entirely.